### DISCUSSION TITLE
updated example showcasing applyChanges()

### DISCUSSION THREAD

**Discussion by Codaea** [bcakmakoglu/vue-flow#1556] [Q&A]
the [confirm delete example](https://vueflow.dev/examples/confirm.html) currently uses the deprecated function `applyNodeChanges()`, instead of `applyChanges()`.

**Comment by bcakmakoglu** [comment]
Thanks for the hint. Ignore the deprecation warning for now 😅

**Reply by Codaea** [reply]
Haha, made it because I couldn't figure out how to use the updated version, figured it out though.
